hippynn/graphs/nodes/base/definition_helpers.py

```diff
@@ -28,7 +28,7 @@ class AutoKw:
     _auto_module_class = NotImplemented
 
     def auto_module(self):
-        kw = self.module_kwargs or {}  # Default to empty dictionary if Falsey
```

Comment:
I know this is way zoomed in on the details, but this is not a typo. "x is truthy" means bool(x) is True, "x is falsey" means "bool(x) is False"

Reply:
Ah, I did not know this.